package.json

```diff
@@ -32,80 +32,80 @@
     "dedent": "^0.7.0",
     "dotparser": "^1.0.0",
     "enumify": "^2.0.0",
-    "graphiql": "^1.4.1",
-    "graphql": "^15.5.0",
-    "graphql-tag": "^2.12.4",
-    "jshint": "^2.12.0",
```

Comment:
Found it weird that it was in our dependencies, and that I also had never noticed it before. It would be used for building the project (i.e. devDependency) but I couldn't find any place where it was used, so removing :+1:  (had a look at blame, it was introduced long ago for the first graph view version I think).